server/db/model/Application.js

```diff
@@ -1,4 +1,41 @@
-/* eslint-disable */
-var mongoose = require('../index.js'),
-    config = require('../../../config/default.js'),
-    Schema = mongoose.Schema;
+var mongoose = require('../index.js'), User = require('./User.js');
+
+// Define the document Schema
+var schema = new mongoose.Schema({
+    user: String,
+    experience: {
+        type: String,
+        enum: ['veteran', 'novice']
+    }
```

Comment:
i don't mind if the profile doesn't enforce them but we need them when someone applies to an event. How exactly that's handled is up to you (although easiest option to me sounda like enforcing required in the profile itself).